www/getChallenge.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+use SimpleSAML\Module\perun\DatabaseConnector;
+use SimpleSAML\Logger;
+
+$entityBody = file_get_contents('php://input');
+$body = json_decode($entityBody, true);
+
+if ($body === false) {
+    Logger::error('Perun.getChallenge: Received invalid json.');
+    http_response_code(400);
+    exit;
+}
+
+if (empty($body['id'] || strlen($body['id']) > 30 || !ctype_print($body['id']))) {
+    Logger::error('Perun.getChallenge: Invalid id');
+    http_response_code(400);
+    exit;
+}
+
+if (empty($body['scriptName'] || strlen($body['scriptName']) > 255 || !ctype_print($body['scriptName']))) {
```

Comment:
typo, missing `)` after `empty($body['scriptName']`
and extra `)` at the end of the line
(this is probably valid code but does something completely unintended)